src/setupProxy.js

```diff
@@ -0,0 +1,14 @@
+const { createProxyMiddleware } = require('http-proxy-middleware');
+
+const apiURL = process.env.REACT_APP_CIQ_API_URL
```

Comment:
I would change the naming convention and remove the "REACT_APP" prefix.

Reply:
Me too, but looks like it's a requirement for REACT to use env vars with "REACT_APP_" prefix...